bridge-adapters/kinetic-bridgehub-adapter-cherwell/instruction.yml

```diff
@@ -0,0 +1,88 @@
+## YAML Template.
+# File structure
+#  Sections = Structures
+#  title Scalar is the sections title.
+#  description Scalar is the text associated with that section.
+#  configs Scalar is mapped to Sequences. The lists are relative to the sections. 
+#   list items have a name Scalar and a description Scalar.
+#  name Scalar is name of the configs property.
+#  notes Scalar is mapped to Sequences. It's additional text added to a section
+#  example Scalar is mapped to Sequences. Each element in the sequence has a 
+#  name and value
+#  value Scalar is a primitive
+---
+title: Overview
+description: >
+  This article covers the configuration of the harvest bridge adapter.
```

Comment:
This file looks like it was just copied from the Harvest bridge adapter. Update for Cherwell.

Reply:
I'll remove instruction.yml because I'm not sure it is even used anymore.